src/components/SourceItem/SourceItem.tsx

```diff
@@ -0,0 +1,145 @@
+/* eslint-disable new-cap */
+import Plausible from 'plausible-tracker';
+import { Grid, Paper, Link, Box } from '@mui/material';
+import Typography from '../Typography';
+import React from 'react';
+import SupportIndicator from '../SupportIndicator/SupportIndicator';
+
+interface SourceProps {
+  claim: Claim;
+  /* The source supporting/rejecting its corresponding claim */
+  source: Source;
+  onFeedbackSourceRelevance?: OnFeedbackSourceRelevance;
+  onFeedbackSourceSupport?: OnFeedbackSourceSupport;
+}
+
+interface DisplayLinkProps {
+  /* The link displayed in the UI */
+  displayLink: string;
+  /* The full URL */
+  link: string;
+}
+
+/* A component to display the link to the article and its publishing date */
+const DisplayLink: React.FC<DisplayLinkProps> = ({ displayLink, link }) => {
+  const { trackEvent } = Plausible();
+
+  const onLinkClicked = () => {
+    trackEvent('Button: Link to source');
+  };
+
+  return (
+    <Grid
+      position="relative"
+      container
+      alignItems="center"
+      spacing={1}
+      justifyContent="center"
+      className="DisplayLink"
+    >
+      <Grid item sx={{ position: 'absolute' }}>
+        <img
+          height={50}
+          width={50}
+          className={'Favicon'}
+          src={'https://www.' + displayLink + '/favicon.ico'}
+          alt={'favicon ' + displayLink}
+          // If no favicon was found then hide the element
+          onError={(i) =>
+            ((i.target as HTMLImageElement).style.display = 'none')
+          }
+          onClick={onLinkClicked}
+        />
+      </Grid>
+    </Grid>
+  );
+};
+
+/*
+ * Displays a single source
+ */
+export const SourceItem: React.FC<SourceProps> = ({ source }) => {
+  // TODO: #98 Implement custom support indicator
+
+  const clampText = (text: string, maxChars: number) => {
+    if (text.length >= maxChars) return text.slice(0, maxChars) + '...';
+    return text;
+  };
+
+  return (
+    <Box sx={{ margin: 18 }}>
+      <Paper elevation={1}>
+        <Grid container direction="column" className="SourceItem" spacing={2}>
+          <Grid item>
+            <DisplayLink
+              displayLink={source.domain ?? ''}
+              link={source.url ?? ''}
+            />
+          </Grid>
+          {source.publishDate != '' && source.publishDate != undefined && (
+            <Grid item>
+              <Typography variant="subtitle1" fontFamily="Roboto">
```

Comment:
FontFamily should be Arizona Serif, I think this is an error in the Figma design. Roboto is the default MUI font but we only want to use Arizona Serif and DM Mono.